### PR TITLE
Mentors!!

### PR DESCRIPTION
**Pull request by SunsetStationuser**
[Changelogs]: 
:cl: Ported over mentors from OracleStation (Credit goes to Ike709)
add: Mentors
code: All the mentor code required
config: There are some Mentor Configs
/:cl:

[why]: # We need to port over mentors! Extremely useful.

### CODE REVIEW

**Comment by SunsetStationuser** [comment]
NEED TO MODULARIZE DO NOT MERGE!!

**Review by ghost — CHANGES_REQUESTED** [review]
I can fix this up tomorrow, cuz i'll be like, trapped inside and shit.

**Comment by FlattestGuitar** [comment]
* This doesn't add any way to assign mentor status to people short of going into the SQL DB. I added that in a later PR, you can go find it on the Oracle github.

* From experience, the mentor follow verb is pure garbage and gets massively in the way of admins who click the link. Either make it print a regular follow link for admins or outright give mentors aghost. I'd opt for the latter, though that might be an issue if mentors aren't completely trustworthy folks.

**Comment by Quantum-M** [comment]
I agree, the Oracle mentor follow was utter garbage, and the only get out of it is via a kindof unclear verb under the mentor tab

File: SQL/tgstation_schema.sql [hunk 1]
```diff
@@ -463,6 +463,20 @@ $$
...
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+DROP TABLE IF EXISTS `mentor_memo`;
+CREATE TABLE `mentor_memo` (
+  `ckey` varchar(32) NOT NULL,
+  `memotext` text NOT NULL,
+  `timestamp` datetime NOT NULL,
+  `last_editor` varchar(32) DEFAULT NULL,
+  `edits` text,
+  PRIMARY KEY (`ckey`)
```
Comment (Birdtalon) [inline SQL/tgstation_schema.sql:478]: Is this going to have multiple entries for each mentor?

Comment (SunsetStationuser) [reply SQL/tgstation_schema.sql:478]: not sure what you are asking here

**Pull request closed without merging**